recipes/mosquitto/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+    "2.0.3":
+      folder: "all"
```

```suggestion
      folder: "2.x"
```
eventually we will add 1.x